src/NuGet.Clients/NuGet.PackageManagement.UI/LoadingStatusIndicator.cs

```diff
@@ -81,5 +76,36 @@ public void SetError(string message)
             Status = LoadingStatus.ErrorOccurred;
             ErrorMessage = message;
         }
+
+        internal string LocalizedStatus
```

Comment:
considering the priority of this issue I am expecting a change to this implementation but it in the future we can consider implementing a derived type of [EnumConverter](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.enumconverter?view=netcore-3.1) to convert enum values to the localized strings.

Reply:
I've been trying to use a IValueConverter but, as we have some custon logic and a custom `LoadingMessage`, I preferred to write a custom method. I can't see the value of using EnumConverter in this case.